### PR TITLE
bz1900784: added control plane nodes 3

### PR DESCRIPTION
**Pull request by nalhadef**
For version 4.6 only (upgrading from 4.5)

Bug: [BZ1900784](https://bugzilla.redhat.com/show_bug.cgi?id=1900784)

Direct link: https://deploy-preview-38885--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned
SME: @oarribas
QE: @jinyunma

### CODE REVIEW

**Comment by nalhadef** [comment]
@oarribas, @jinyunma: Please confirm which versions of OCP this upgrade from 4.5 information needs to be added to.

**Comment by jinyunma** [comment]
@nalhadef Issue only happened when upgrading from 4.5 to 4.6 version until now

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -138,6 +139,8 @@ An additional role is required if the installation program is to create a vSpher
 `VirtualMachine.Inventory.CreateFromExisting`
 `VirtualMachine.Inventory.Delete`
 `VirtualMachine.Provisioning.Clone`
+`VirtualMachine.Provisioning.DeployTemplate`
+`VirtualMachine.Provisioning.MarkAsTemplate`
```
Comment (jinyunma) [inline modules/installation-vsphere-installer-infra-requirements.adoc:143]: Both permissions only need to be added on 4.8+ since ova images will be convert to template from ocp4.8 once it is uploaded to vsphere.

File: modules/installation-vsphere-installer-infra-requirements0.adoc [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+default. This record must be resolvable by both clients external to the cluster
+and from all the nodes within the cluster.
+|===
+
+ifeval::["{context}" == "installing-restricted-networks-installer-provisioned-vsphere"]
+:!restricted:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-vmc"]
+:!restricted:
+endif::[]
```
Comment (jinyunma) [inline modules/installation-vsphere-installer-infra-requirements0.adoc:299]: what's the purpose of this update? Seems unrelated with this PR.

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 1 above)
Comment (nalhadef) [reply modules/installation-vsphere-installer-infra-requirements.adoc:143]: Lines 142 and 143 removed.

File: modules/installation-vsphere-installer-infra-requirements0.adoc (hunk 2 above)
Comment (nalhadef) [reply modules/installation-vsphere-installer-infra-requirements0.adoc]: Unwanted file removed.

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 3]
```diff
@@ -72,6 +72,7 @@ An additional role is required if the installation program is to create a vSpher
 `Datastore.AllocateSpace`
 `Datastore.Browse`
 `Datastore.FileManagement`
+`InventoryService.Tagging.ObjectAttachable`
```
Comment (jinyunma) [inline modules/installation-vsphere-installer-infra-requirements.adoc:75]: I checked that permission is added starting from 4.9, it would not be added here on 4.6.

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 1 above)
Comment (jinyunma) [reply modules/installation-vsphere-installer-infra-requirements.adoc:143]: I can still see 2 lines in doc from direct link

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 3 above)
Comment (nalhadef) [reply modules/installation-vsphere-installer-infra-requirements.adoc:75]: Removed
